prod/westeurope/internal/api/functions_public/function_app/terragrunt.hcl

```diff
@@ -63,6 +63,8 @@ inputs = {
     StorageConnection = dependency.storage_account.outputs.primary_connection_string
 
     VALIDATION_CALLBACK_URL = "https://app-backend.io.italia.it/email_verification.html"
+
+    SLOT_TASK_HUBNAME = "ProductionTaskHub"
```

Comment:
ditto (actually the name of the task hub is default: https://github.com/pagopa/io-functions-public/blob/master/host.json#L13)

Reply:
at this point it may be worth to upgrade to runtime v3 making a new function app then switch (for fn-admin and fn-public)